web/pdf_find_controller.js

```diff
@@ -359,23 +360,23 @@ class PDFFindController {
 
     this.active = true;
 
-    if (this.dirtyMatch) {
+    if (this._dirtyMatch) {
       // Need to recalculate the matches, reset everything.
-      this.dirtyMatch = false;
+      this._dirtyMatch = false;
       this.selected.pageIdx = this.selected.matchIdx = -1;
-      this.offset.pageIdx = currentPageIndex;
-      this.offset.matchIdx = null;
-      this.resumePageIdx = null;
+      this._offset.pageIdx = currentPageIndex;
+      this._offset.matchIdx = null;
+      this._resumePageIdx = null;
       this.pageMatches = [];
```

Comment:
This is pre-existing, but it might be a tiny bit more efficient to do `this.pageMatches.length = 0;` here instead, since that would re-use the existing array rather than creating a new one.

Reply:
Done in the new commit series.